gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -208,5 +211,25 @@ export const wrapCommand = (command) => {
       logger.info(`Eligible owners to sign: `)
       logger.info(remainingEligibleSigners.toString())
     }
+
+    simulateTX = async (tx) => {
+      logger.loading(`Simulating proposal`)
+      const simulationResult = await this.provider.simulate(tx)
+      if(simulationResult.value.err) {
+        logger.error(`Failed to simulate proposal.`)
+        const customError = simulationResult.value.err['InstructionError'][1]['Custom']
+        if (customError) {
+          const { idl } = getContract(command.id.split(':')[0], '')
```

Comment:
This should be tackled in https://github.com/smartcontractkit/chainlink-solana/issues/127